src/main/java/gov/nci/ppe/services/impl/UserServiceImpl.java

```diff
@@ -196,6 +196,9 @@ public UserServiceImpl(UserRepository userRepo, CodeRepository codeRepo, Partici
 				user.setPortalAccountStatus(codeRepository.findByCodeName(PortalAccountStatus.ACCT_ACTIVE.name()));
 				user = userRepository.save(user);
 				optionalUser = Optional.of(user);
+			} else if (user.getPortalAccountStatus().getCodeName()
+					.equals(PortalAccountStatus.ACCT_TERMINATED_AT_PPE.name())) {
```

Comment:
What about when we log in? Don't we need to validate during login?

Reply:
This is called from login